### ISSUE TITLE
Replace label in add-github-handle.md

### ISSUE THREAD

**Issue by ExperimentsInHonesty** [hackforla/website#7600]
### Overview
We need to change the ready for label on the add-github-handle.md issue template so that the merge team can handle more of the issue reviews.

### Action Items
- [x] Open the files  /.github/ISSUE_TEMPLATE/add-github-handle.md
- [x] In the following text, replace `ready for dev lead` label with `ready for merge team` label 
```
labels: 'Complexity: Small, feature: spelling, good first issue, P-Feature: Project
  Info and Page, ready for dev lead, role: back end/devOps, role: front end, size:
  0.5pt, size: 0.25pt'
```
- [x] Push your issue branch to your fork of the website repository, as described in [Section 3.1.a of CONTRIBUTING doc](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#31a-push-all-changes-to-your-issue-branch), for example:  `git push --set-upstream origin test-update-labels-1234`  
- [x] The response from the push command will include the URL for creating a pull request. Copy/Paste the URL for creating the Pull Request for later.  
- [x] Browse to your fork of the website repository and click Settings 
  <details><summary>Screenshot of Repository Settings</summary>

  ![image](https://github.com/hackforla/website/assets/5314153/8ecf2e55-2187-4ab9-acf1-2fd7548291a1)

  </details> 
- [x] Change the default branch to the new issue branch
  <details><summary>Screenshot of Default Branch Setting</summary>

  ![image](https://github.com/hackforla/website/assets/5314153/945fe02e-035f-4215-9c25-1d46c209d8c0)

  </details> 
- [x] Further down on the settings page under Features, check the Issues checkbox
  <details><summary>Screenshot of Issues Checkbox</summary>

  ![image](https://github.com/hackforla/website/assets/16524851/563fc427-ba07-4e47-9440-2580bbfcad52)

  </details> 

- [x] From the Issues page, click "New Issue" then locate the template updated by your issue, and click "Get Started" to open the issue template. 
- [x] Copy the URL of the issue creation page and save for later use.  
- [x] Verify the appearance of the issue template.
- [x] Complete the Pull Request process by visiting the URL saved from the push command.
- [x] In the PR copy/paste this text immediately before the section "Screenshots of Proposed Changes Of The Website"
```
 ### For Reviewers
- Use this URL to check the updated issue template: [INSERT URL of issue creation page]
```
- [x] Replace the text in the square brackets (as well as the square brackets) with the URL of the updated issue template in your fork of the repository
- [x] Submit your pull request with your changes for review
- [x] After PR is merged, be sure to follow the steps above to change your default branch back to `gh-pages` and to disable Issues feature

### Resources/Instructions
For QA to validate change: https://github.com/hackforla/website/blob/gh-pages/.github/ISSUE_TEMPLATE/add-github-handle.md?plain=1

- This issue is part of Issue Making Issue #7602

**Comment by HackforLABot** [comment]
Hello @siyunfeng, we appreciate you taking on this issue, however it looks like you're already working on another issue at this time. Please wait until your current issue is merged before taking on another issue. :)

We are going to unassign you from this issue so you can focus on your current issue.

Hfla appreciates you! :)

**Comment by HackforLABot** [comment]
Hi @siyunfeng, thank you for taking up this issue! Hfla appreciates you :)

Do let fellow developers know about your:-
i. **Availability:** (When are you available to work on the issue/answer questions other programmers might have about your issue?)
ii. **ETA:** (When do you expect this issue to be completed?)

You're awesome!

P.S. - You may not take up another issue until this issue gets merged (or closed). Thanks again :)

**Comment by siyunfeng** [comment]
**Availability**: Weekdays (6-10 PM, EDT)
**ETA**: Thursday 10/31/2024

**Comment by siyunfeng** [comment]
Hi, @ExperimentsInHonesty !

I want to clarify what do I need to do with `Verify the appearance of the issue template.`?

Does it mean manually editing and adding all the labels from the updated file `/.github/ISSUE_TEMPLATE/add-github-handle.md` to the issue template? Because I don't see those labels in the URL I provided to the reviewers in my PR. The only label shown in the issue template I created is `good first issue`. 

- Original issue template: 

![labels-section-in-the-original-issue-template](https://github.com/user-attachments/assets/32d41dc9-c1cb-4f3b-8318-9d17f9071cbd)

- New issue template:

![labels-section-in-the-new-issue-template](https://github.com/user-attachments/assets/2af7881e-5172-4e65-b57e-a17e53a981a7)
